### ISSUE TITLE
Data missing/wrong - authentication not used

### ISSUE THREAD

**Issue by Datafreak** [fetzerch/fritzcollectd#22]
Hi,

I have a FRITZ!Box 6490 Cable (lgi) with FRITZ!OS 06.88 from Unitymedia.
Since they have updates the OS a lot of data will be not collected anymore.
I had created a user since the first use. So the authentication should be used.

I'm most interested at the constatus and dslstatus. But it is since the update only "0".
I think something about the authentication is not working.

When I'm using the fritzstatus from fritzconnection without authentication, also the data are missing/wrong:
```
~$ fritzstatus

FritzStatus:
version:            0.5.1
model:              FRITZ!Box 6490 Cable (lgi)
is linked:          False
is connected:       False
external ip:        109.193.13.221
uptime:             00:00:00
bytes send:         0
bytes received:     0
max. bit rate:      ('10.0 MBit/s', '214.8 MBit/s')
```

When I used the [patched Version](https://bitbucket.org/kbr/fritzconnection/pull-requests/14/adds-authentication-support-to-fritzstatus/diff) from [JayPiKay](https://bitbucket.org/JayPiKay/fritzconnection/branch/status_auth) it is working:
```
~$ fritzstatus -u fritz-config -p XXXXXXXXXXXXXXXXXXXXXXXXXX

FritzStatus:
version:            0.5.2
model:              FRITZ!Box 6490 Cable (lgi)
is linked:          True
is connected:       True
external ip:        109.193.13.221
uptime:             31:48:37
bytes send:         0
bytes received:     0
max. bit rate:      ('10.0 MBit/s', '214.8 MBit/s')
```

My collectd.conf is looking like this:
```<Plugin python>
        ModulePath "/usr/local/lib/python2.7/dist-packages"
#       LogTraces true
#       Interactive true
#       Import "spam"
        Import "fritzcollectd"

        <Module fritzcollectd>
                Address "192.168.1.1"
                Port 49000
                User "fritz-config"
                Password "XXXXXXXXXXXXXXXXXXXXXXXXXX"
                Hostname "FritzBox"
                Instance "1"
                Verbose "False"
        </Module>
#
#       <Module spam>
#               spam "wonderful" "lovely"
#       </Module>
</Plugin>
```

I already have tried to debug this issue in your code, but I was not successful.
Is there any tip you can give me to find the reason for this?

Best Regards,
Datafreak

**Comment by Datafreak** [comment]
Hi @fetzerch,

thanks for the tip. I have changed the line and now I get the constatus and dslstatus again.

Best Regards,
Thomas

**Comment by carlolf** [comment]
Because my influxdb table `'fritzbox' `still has no data, I tried this approach here with supplying the fritzbox **password** in the script. I also changed fritzbox's config according to require user and password on the connection. But no success. The output of `'fritzstatus.py'` still says `'is_linked: False'` and `'is_connected: False'`. 

Any idea? My Fritzbox is a 6490 from KDG.

**Comment by fetzerch** [comment]
The change mentioned in https://github.com/fetzerch/fritzcollectd/issues/22#issuecomment-431575511 only works for `fritzcollectd`. `fritzstatus.py` is a different tool (from `fritzconnection`), you'd have to adapt it here as well to get it working.

See https://bitbucket.org/kbr/fritzconnection/pull-requests/14/adds-authentication-support-to-fritzstatus/diff for a proposed change that hasn't yet made it into `fritzconnection`.

**Comment by Doxylamin** [comment]
Thanks to your tip as well, although I now receive constatus and dslstatus, it's still not showing me total dl/ul, last 24h dl/ul and traffic stats. Any tips on that one? Using the same model and version as the OP.

**Comment by Datafreak** [comment]
Since the Update to [FRITZ!OS: 07.01](https://stadt-bremerhaven.de/fritzos-7-01-fuer-fritzbox-6490-bei-unitymedia/) everything is working again.
So AVM or Unitymedia had an Bug in FRITZ!OS 06.88 and fixed it with FRITZ!OS: 07.01.
